src/util/util_string.h

```diff
@@ -38,4 +38,33 @@ namespace dxvk::str {
         append(stream, args...);
         return stream.str();
     }
+
+    std::string fromnullable(const char* str);
```

Comment:
Would be nice to add a few normal unit tests for the new string utils here https://github.com/jp7677/dxvk-nvapi/blob/master/tests/util.cpp#L37